src/index.html

```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div
+    class="calendar
+    calendar--month-length-29
```

Comment:
are you sure your calendar work correct?
if you change ` calendar--start-mon` to ` calendar--start-sun` the calendar should change and display correctly. 
try it